schemas/com.mailgun/message_bounced/jsonschema/1-0-0

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message bounced event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_bounced",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "attachment-count": {
+      "description": "Number of attachments in attachment-x (where x is a number) fields.",
+      "type": "integer",
+      "maximum": 32767
+    },
+    "body-plain": {
+      "description": "text/plain body of the message.",
+      "type": "string",
+      "maxLength": 524288
+    },
+    "event": {
+      "description": "Event name ('bounced').",
+      "type": "string",
+      "maxLength": 16
```

Comment:
Are we sure about such short length? Can't it be `enum` instead?

Reply:
I thought about enum but:
a. This is a constant (i.e. it will always be 'bounced' in this case, so we may as well omit it. 
b. Enums have to be updated to add a value. 

The length is certainly sufficient.

Reply:
Yeah, I agree. `enum`s are good, but need be used only when we're 100% sure.

Reply:
Yes, enums are fairly risky when we don't control the original data source (i.e. all webhooks)...